src/lm_buddy/buddy.py

```diff
@@ -0,0 +1,41 @@
+from lm_buddy.integrations.wandb import ArtifactLoader, WandbArtifactLoader
+from lm_buddy.jobs._entrypoints import run_finetuning, run_lm_harness, run_prometheus, run_simple
+from lm_buddy.jobs.common import EvaluationResult, FinetuningResult, SimpleResult
+from lm_buddy.jobs.configs import (
+    EvaluationJobConfig,
+    FinetuningJobConfig,
+    LMHarnessJobConfig,
+    PrometheusJobConfig,
+    SimpleJobConfig,
+)
+
+
+class LMBuddy:
```

Comment:
Per our discussions over Zoom, we decided that it would be difficult to built a single workflow that runs all tasks consecutively at the moment. However, I do feel it is nicer to have this class that can dispatch to the various jobs, even if it doesn't support running things in a pipeline atm.

So for now, I am adding this as a thin wrapper around the entrypoint functions, and returning explicit return types for the data generated by the jobs.